rocky/reports/views/report_overview.py

```diff
@@ -92,6 +92,25 @@ def get_queryset(self) -> list[dict[str, Any]]:
 
         return recipes
 
+    def post(self, request, *args, **kwargs):
```

Comment:
👌 

I think this is a bit simpler now because it's a more "compact" API and it's easy to find the delete logic!